dotCMS/src/integration-test/java/com/dotcms/vanityurl/business/VanityUrlAPITest.java

```diff
@@ -73,6 +72,65 @@ public static void prepare() throws Exception {
     }
 
     
+    /**
+     * tests that we correctly order the cached vanites.  This ordering is done
+     * by the findInDb method, which should return a hosts' vanities in order asc
+     * 
+     * It also tests that the vanityUrlAPI.resolveVanityUrl returns
+     * the first matching vanity in the list
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void testing_vanityurl_ordering() throws Exception {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6064444)